src/main/java/de/tum/in/www1/artemis/domain/leaderboard/tutor/TutorLeaderboardComplaintResponses.java

```diff
@@ -28,19 +24,19 @@ public long getUserId() {
         return userId;
     }
 
-    public long getCourseId() {
-        return courseId;
+    public TutorLeaderboardComplaintResponses(long userId, long complaintResponses, double points) {
+        this.userId = userId;
+        this.complaintResponses = complaintResponses;
+        this.points = points;
     }
 
-    public long getExerciseId() {
-        return exerciseId;
+    public TutorLeaderboardComplaintResponses() {
+        this.userId = 0L;
+        this.complaintResponses = 0L;
+        this.points = 0.0;
```

Comment:
i guess you could use 0D instead of 0.0 here.

```suggestion
        this.points = 0D;
```